chain-impl-mockchain/tests/common/arbitrary/transaction.rs

```diff
@@ -0,0 +1,45 @@
+use crate::common::address::AddressData;
+use chain_addr::Address;
+use chain_impl_mockchain::transaction::Output;
+use chain_impl_mockchain::value::Value;
+use quickcheck::{Arbitrary, Gen};
+
+#[derive(Clone, Debug)]
+pub struct ArbitraryValue(pub Value);
+
+impl Arbitrary for ArbitraryValue {
+    fn arbitrary<G: Gen>(gen: &mut G) -> Self {
+        ArbitraryValue(Value(u64::arbitrary(gen)))
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct ArbitraryOutput(pub Output<Address>);
+
+impl Arbitrary for ArbitraryOutput {
```

Comment:
Same as above, no need to wrap these types at all if the property test API is implemented in the library crate.